### ISSUE TITLE
ReleaseList is expecting `count` and `offset` but is provided `release-count` and `release-offset`

### ISSUE THREAD

**Issue by seankearney** [avatar29A/MusicBrainz#52]
The test `TestReleaseBrowseAsync()` in [my fork](https://github.com/seankearney/MusicBrainz/commit/d8de817443789bad2dc10ac3dd645d74c59cd795) demonstrates that there is a bug with `ReleaseList.Count` (and undemonstrated is an issue with `Offset`).

The URL https://musicbrainz.org/ws/2/release?artist=c3cceeed-3332-4cf0-8c4c-bbde425147b6&limit=40&offset=0&inc=&fmt=json will return this:

```
{
  "releases": [
    {
...
    }
  ],
  "release-count": 448,
  "release-offset": 0
}
```

The code assumes it has `count` and `offset` (without the `release-` prefix).

**Comment by wo80** [comment]
I filed an issue for this a while ago: https://tickets.metabrainz.org/browse/MBS-9731

Fixed this in the [develop](https://github.com/avatar29A/MusicBrainz/commit/c0791e82a522fce66d993c46c8350d1ecedc0287) branch, though I don't like the solution of hiding the base members.

**Comment by seankearney** [comment]
Thanks for the quick reply @wo80. 

I have the same quick fix that addresses the `ReleaseList` problem for my use case, but you have a much more comprehensive solution in `develop`.

Curious if you have a timeline for the next NuGet release that would include this fix? I can maintain my own fork and private nupkg, but if you are nearing a release I'd prefer to stay "official".

Thanks!

**Comment by wo80** [comment]
Would a prerelease package of the develop branch be okay?

I'm still trying to figure out the best way, how to add `CancellationToken` support. That was what kept me from merging #44, but I guess there is no "nice" way with the current API design.

**Comment by seankearney** [comment]
If you have a prerelease version of the package I'm happy to use/test it and provide feedback. My usage is rather limited though.

**Comment by wo80** [comment]
I have published the prerelease package [2.2.0-pre-1](https://www.nuget.org/packages/MusicBrainzAPI/2.2.0-dev-1). The version is still 2.x since there are no breaking changes at the moment, but you should get deprecation warnings regarding the static API.

Please let me know if there are any problems compiling your code. The plan is be backward compatible until version 3 comes out.

**Comment by seankearney** [comment]
I swapped references from my private build to this and it worked straight away. Compiler warnings as expected, but no code changes required and worked as expected.

Thanks for releasing this!
